rafthttp/peer_status.go

```diff
@@ -47,6 +52,13 @@ func (s *peerStatus) activate() {
 		plog.Infof("peer %s became active", s.id)
 		s.active = true
 		s.since = time.Now()
+
+		if s.once != nil {
+			s.once.Do(func() {
+				plog.Infof("notifying of active peer %q", s.id)
```

Comment:
this logging does not seem to be useful.

Reply:
Removed.